code/modules/mob/language/codespeak.dm

```diff
@@ -0,0 +1,140 @@
+	set name = "MAYDAY"
+	src.codesay("Need help", TRUE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_clear()
+	set category = "Codespeak"
+	set name = "Area clear"
+	src.codesay("No hostiles", TRUE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_romch()
+	set category = "Codespeak"
+	set name = "Roaches"
+	src.codesay("Roaches", TRUE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_bigromch()
+	set category = "Codespeak"
+	set name = "Fuhrer roach"
+	src.codesay("Fuhrer roach", TRUE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_serb()
+	set category = "Codespeak"
+	set name = "Serbs"
+	src.codesay("Serbian mercs", TRUE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_commie()
+	set category = "Codespeak"
+	set name = "Excels"
+	src.codesay("Excelsior infiltrators", TRUE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_carrion()
+	set category = "Codespeak"
+	set name = "Carrion"
+	src.codesay("Carrion presence", TRUE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_mutant()
+	set category = "Codespeak"
+	set name = "Mutant"
+	src.codesay("Unsanctioned organism", TRUE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_dead()
+	set category = "Codespeak"
+	set name = "Dead guy"
+	src.codesay("Crewmember flatlined", TRUE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_corpse()
+	set category = "Codespeak"
+	set name = "Wounded guy"
+	src.codesay("Crewmember wounded", TRUE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_criminal()
+	set category = "Codespeak"
+	set name = "Suspect"
+	src.codesay("Located the suspect", TRUE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_status()
+	set category = "Codespeak"
+	set name = "Status?"
+	src.codesay("What's the status?", FALSE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_shutup()
+	set category = "Codespeak"
+	set name = "Shut up"
+	src.codesay("Unnecessary use of radio", FALSE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_understood()
+	set category = "Codespeak"
+	set name = "Ok"
+	src.codesay("Affirmative", FALSE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_yes()
+	set category = "Codespeak"
+	set name = "Yes"
+	src.codesay("Positive", FALSE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_no()
+	set category = "Codespeak"
+	set name = "No"
+	src.codesay("Negative", FALSE, FALSE)
+
+/mob/living/carbon/human/proc/codespeak_understood_local()
+	set category = "Codespeak"
+	set name = "(local) Ok"
+	src.codesay("Affirmative", FALSE, TRUE)
+
+/mob/living/carbon/human/proc/codespeak_yes_local()
+	set category = "Codespeak"
+	set name = "(local) Yes"
+	src.codesay("Positive", FALSE, TRUE)
+
+/mob/living/carbon/human/proc/codespeak_no_local()
+	set category = "Codespeak"
+	set name = "(local) No"
+	src.codesay("Negative", FALSE, TRUE)
+
+/mob/living/carbon/human/proc/codespeak_warcrime_local()
+	set category = "Codespeak"
+	set name = "(local) Attack?"
+	src.codesay("We attack?", FALSE, TRUE) //TODO: Replace with something that sounds good
```

Comment:
Maybe `Engage?` or `Engage target?`

Reply:
Tried that, looks wanky too. But all better than "attack" I guess. Replaced.